server.js

```diff
@@ -1,9 +1,11 @@
 var express = require('express');
 var app = express();
-
+var bodyParser = require('body-parser');
 var json2csv = require('json2csv');
 var fs = require('fs');
 
+app.use(bodyParser.json());
+
 var csvFields = ['correct', 'duration', 'real', 'response', 'responseTime', 'state', 'word'];
```

Comment:
if we add additional items to this array, will this automatically update the fields it outputs or does JSON2CSV have some hard coded stuff in it that we need to update as well. i.e. just a comment. showing where to make changes to the JSON you're updating.